var/spack/repos/builtin/packages/cbtf-krell/package.py

```diff
@@ -75,7 +75,7 @@ class CbtfKrell(CMakePackage):
     depends_on("cmake@3.0.2:", type='build')
 
     # For binutils service
-    depends_on("binutils@2.24+krellpatch")
+    depends_on("binutils")
```

Comment:
@jgalarowicz Do these packages actually need the `binutils` dependency at all? See http://spack.readthedocs.io/en/latest/getting_started.html#binutils